Basic-Video-Chat-SPM/README.md

```diff
@@ -0,0 +1,13 @@
+Basic Video Chat sample app
+===========================
+
+The Basic Video Chat app is a very simple application meant to get a new developer
+started using the VonageVideo iOS SDK. 
+
```

Comment:
Since this may be the first sample that the user can open maybe it will be important to add what we have for Basic Video Chat README

`The Basic Video Chat app is a very simple application meant to get a new developer
started using the OpenTok iOS SDK. For a full description, see the [Basic tutorial at the
OpenTok developer center](https://tokbox.com/developer/tutorials/ios/basic-video-chat/).`

Reply:
I was thinking to point that tutorial, but it makes reference to Cocoapods, maybe we can request an update on that and the reference from this sample app.